src/functions.php

```diff
@@ -75,86 +48,57 @@ function uri_for($uri)
  * - metadata: Array of custom metadata.
  * - size: Size of the stream.
  *
+ * This method accepts the following `$resource` types:
```

Comment:
Not sure we need to add this on a deprecated function

Reply:
I took the view that the functions and static methods should have identical phpdoc.